app/src/main/AndroidManifest.xml

```diff
@@ -33,7 +33,36 @@
 
 				<category android:name="android.intent.category.LAUNCHER" />
 			</intent-filter>
+            <intent-filter android:autoVerify="true">
```

Comment:
Make sure the indentation here is consistent, please!

Reply:
Updated indentation for the whole file. It used tabes before.